magenta-lib/src/main/scala/magenta/tasks/ChangeSetTasks.scala

```diff
@@ -0,0 +1,150 @@
+package magenta.tasks
+
+import com.amazonaws.services.cloudformation.model.{ChangeSetType, DeleteChangeSetRequest, DescribeChangeSetRequest, ExecuteChangeSetRequest}
+import com.amazonaws.services.s3.AmazonS3
+import magenta.artifact.S3Path
+import magenta.deployment_type.CloudFormationDeploymentTypeParameters.{CfnParam, TagCriteria}
+import magenta.tasks.CloudFormation.ParameterValue
+import magenta.tasks.UpdateCloudFormationTask._
+import magenta.{DeployReporter, KeyRing, Region, Stack, Stage}
+
+import scala.collection.JavaConverters._
+
+case class CreateChangeSetTask(
```

Comment:
This is a controversial use of `case class`, these are behaviour not just data structures so I'd expect them to be vanilla classes.
I assume this is for consistency with the rest of the codebase in which case no worries?

Reply:
Agreed. We'll make them classes (no need to copy existing odd practice)